<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7VCDQC7" failures="0" tests="4" name="api.test.UsersDD" time="7.387" errors="1" timestamp="2023-12-04T21:49:27 GMT-06:00" skipped="0">
  <testcase classname="api.test.UsersDD" name="createUsers" time="0.353"/>
  <testcase classname="api.test.UsersDD" name="createUsers" time="0.329"/>
  <testcase classname="api.test.UsersDD" name="createUsers" time="5.433"/>
  <testcase classname="api.test.UsersDD" name="getUsers" time="1.272">
    <error message="
[public void api.test.UsersDD.getUsers(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: getUsers([java.lang.String arg0])
Arguments: [(java.lang.String) 71706943,(java.lang.String) SurajM,(java.lang.String) Suraj,(java.lang.String) Mishra,(java.lang.String) mishraskus1@gmail.com,(java.lang.String) test1,(java.lang.String) 1112223334]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void api.test.UsersDD.getUsers(java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: getUsers([java.lang.String arg0])
Arguments: [(java.lang.String) 71706943,(java.lang.String) SurajM,(java.lang.String) Suraj,(java.lang.String) Mishra,(java.lang.String) mishraskus1@gmail.com,(java.lang.String) test1,(java.lang.String) 1112223334]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- getUsers -->
</testsuite> <!-- api.test.UsersDD -->
